docker/spark/scripts/entrypoint-submit.sh

```diff
@@ -44,7 +44,6 @@ if [[ ! -z ${HIVE_METASTORE_URI} ]]; then
 echo -e "
```

Comment:
You need to use this in the tests instead of writing catalogImplementation and metastore.uris over and over again